scrapers/Hegre.yml

```diff
@@ -29,7 +30,7 @@ xPathScrapers:
       Title: //meta[@property="og:title"]/@content
       Details:
         selector: |
-          //div[@class="film-subtitle"]/text()
+          //div[@class="film-subtitle translated-text"]/text()
```

Comment:
Is this for all scenes?
Would it be safer to use something like
```yaml
          //div[contains(@class,"film-subtitle")]/text()
```

Reply:
That makes more sense, thanks for the suggestion.